bank-balance-exactly-once/src/main/java/com/github/simplesteph/udemy/kafka/streams/BankBalanceExactlyOnceApp.java

```diff
@@ -33,18 +32,13 @@ public static void main(String[] args) {
 
         // Exactly once processing!!
         config.put(StreamsConfig.PROCESSING_GUARANTEE_CONFIG, StreamsConfig.EXACTLY_ONCE);
-
-        // json Serde
-        final Serializer<JsonNode> jsonSerializer = new JsonSerializer();
-        final Deserializer<JsonNode> jsonDeserializer = new JsonDeserializer();
-        final Serde<JsonNode> jsonSerde = Serdes.serdeFrom(jsonSerializer, jsonDeserializer);
-
-
-        KStreamBuilder builder = new KStreamBuilder();
-
-        KStream<String, JsonNode> bankTransactions =
-                builder.stream(Serdes.String(), jsonSerde, "bank-transactions");
-
+        config.put(StreamsConfig.DEFAULT_KEY_SERDE_CLASS_CONFIG, Serdes.serdeFrom(JsonSerializer.class));
```

Comment:
why did you remove the SerDes and add default? I think it's fine, but we want the videos to keep in line with what we already have, so the previous code should be fine

Reply:
I believe I followed the example given here
https://kafka.apache.org/10/documentation/streams/